addons/project/tests/test_project_report.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import tagged
+from .test_project_base import TestProjectCommon
+
+@tagged('post_install', '-at_install')
+class TestProjectReport(TestProjectCommon):
+    def test_avg_rating_measure(self):
+        rating_vals = {
+            'res_model_id': self.env['ir.model']._get('project.task').id,
+            'rated_partner_id': self.partner_1.id,
+            'partner_id': self.partner_1.id,
+            'consumed': True,
+        }
```

Comment:
@xavierbol this PR is ready for final review and merge

Reply:
This PR is not ready for the merge since there is still a fixup commit. ^^
When @IT-Ideas has time, we will do the final review.

Reply:
I will squash changes, thanks